README.md

```diff
@@ -2,13 +2,31 @@
 Listen to music from your Spotify Premium music account
 
 ## About
-Stream your favorite music from the popular Spotify music service.  Spotify
+Stream your favorite music from the popular Spotify music service. Spotify
 Premium users can search and play tracks from their own playlists or the huge
 Spotify music library.
 
 You can also control your Mycroft device using the Spotify Connect system.
 So play DJ on your phone while listening on Mycroft!
 
+### This skill doesn't do any playback
+
+This skills works with the spotify connect protocol and generally requires the user to setup a spotify player themselves.
+
+For picroft users [raspotify](https://github.com/dtcooper/raspotify) is a good choice.
```

Comment:
```suggestion
For Picroft users, [raspotify](https://github.com/dtcooper/raspotify) is a good choice.
```

Reply:
It's worth mentioning that raspotify on its own runs on a seperate user to the pi user, and so can't access Picroft's Pulseaudio instance without config changes. I think we already discussed this once, so sorry if I'm repeating myself :)

Reply:
which "config changes" are you remembering here? - I don't think I recall anything special/extra about it.
(once I understood that raspotify was needed)

Reply:
I edited the service to run under the pi user, rather than the raspotify user.